### PR TITLE
amd.md: Add new section on AMD graphics

### LINKED ISSUE #310: Missing AMD graphics section
- explain `xf86-video-ati` vs `xf86-video-amdgpu`

### PR DESCRIPTION
**Pull request by ahesford**
Fixes #310.

I'm not an expert on graphics hardware or its configuration, but I have a relatively modern AMD card and use the packages mentioned in this document. If anybody else knows more about graphics, maybe we could add a little bit of context.

### CODE REVIEW

File: src/config/graphical-session/graphics-drivers/amd.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+installed the `linux` or `linux-lts` packages, it will be installed as a
+dependency. If you installed a version-specific kernel package (e.g.,
+`linux5.4`), it may be necessary to manually install `linux-firmware-amd`.
+
+## OpenGL
+
+Install the Mesa DRI package, `mesa-dri`.
+
+> Note: This is already included in the `xorg` meta-package, but it is needed
+> when installing Xorg via `xorg-minimal` or for running a Wayland compositor.
```
Comment (ericonr) [inline src/config/graphical-session/graphics-drivers/amd.md:13]: We are removing block quotes from the docs when they aren't quotes, so this should be a normal paragraph.

File: src/config/graphical-session/graphics-drivers/amd.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+## Vulkan
+
+Install `vulkan-loader`, the Khronos Vulkan Loader. Then install one or both of
+the Mesa AMD Vulkan driver, `mesa-vulkan-radeon`; or the GPUOpen AMD Vulkan
+driver, `amdvlk`.
+
+## Xorg
+
+Installing the `xorg` meta-package will pull in both `xf86-video-amdgpu` and,
+for older hardware, `xf86-video-ati`. If you install `xorg-minimal`, choose one
```
Comment (ericonr) [inline src/config/graphical-session/graphics-drivers/amd.md:24]: Do you know of any way to determine which one is necessary?

File: src/config/graphical-session/graphics-drivers/index.md [hunk 3]
```diff
@@ -5,6 +5,7 @@ of your system.
 
 ## Section Contents
 
-- [Intel GPU](./intel.md)
-- [NVIDIA Optimus](./optimus.md)
+- [AMD](./amd.md)
+- [Intel](./intel.md)
 - [NVIDIA](./nvidia.md)
+- [NVIDIA Optimus](./optimus.md)
```
Comment (ericonr) [inline src/config/graphical-session/graphics-drivers/index.md:11]: I like this organization, but the changes should be made in the SUMMARY as well.

Comment (ericonr) [reply src/config/graphical-session/graphics-drivers/index.md:11]: Ideally the title here matches the one in the page, too (I believe it is necessary for the PDF gen). Perhaps "AMD and ATI" ?

**Comment by st3r4g** [comment]
Looks good as a starting point

**Pull request closed without merging**